### PR TITLE
1)Added checkstyle.xml to testing java projects. 2) Added git-commit plugin and travis.yaml. Updated pom.xml 

### PR DESCRIPTION
**Pull request by BasileoLuv**

### CODE REVIEW

**Review by savermyas — CHANGES_REQUESTED** [review]
1) This pull request contains the extra commits that must not be included. Update your fork first.
2) What is the desired result?
```
bash-3.2$ mvn clean package && unzip -l target/Calc-1.0-SNAPSHOT.jar | grep properties
...
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 5.572 s
[INFO] Finished at: 2019-02-08T14:46:35+03:00
[INFO] ------------------------------------------------------------------------
       84  02-08-2019 14:46   META-INF/maven/com.clc/Calc/pom.properties
      909  02-08-2019 14:46   git.properties
       39  02-08-2019 14:46   git-commit-id.properties
```

I see a few properties files with different content which one is needed?

**Comment by BasileoLuv** [comment]
2)we get information about commit from the git.properties
If we need only hash:
sed -n '10p' target/classes/git.properties > target/classes/anyfile.txt

File: README.md [hunk 1]
```diff
@@ -13,7 +13,10 @@ mvn package test
 ```
 java -jar target/Calc-1.0-SNAPSHOT.jar 3 + 5
 ```
-# How to run checkstyle
+# How to package
 ```
-mvn checkstyle:checkstyle -Dcheckstyle.config.location=checkstyle.xml
+mvn package
```
Comment (savermyas) [inline README.md]: We've already got this instruction earlier. Why did you remove the checkstyle instruction?

File: pom.xml [hunk 2]
```diff
@@ -4,14 +4,23 @@
 
     <groupId>com.clc</groupId>
     <artifactId>Calc</artifactId>
-    <version>1.0.2-SNAPSHOT</version>
+    <version>1.0-SNAPSHOT</version>
```
Comment (savermyas) [inline pom.xml]: You must not decrease the development version. Maximum - inctrease.

File: pom.xml [hunk 3]
```diff
@@ -4,14 +4,23 @@
...
     <packaging>jar</packaging>
 
     <name>Calculator</name>
     <url>http://maven.apache.org</url>
+           
+  <distributionManagement>
```
Comment (savermyas) [inline pom.xml]: this section comes from settings.xml
you don't need it in pom.

File: pom.xml [hunk 4]
```diff
@@ -38,6 +73,14 @@
         </archive>
       </configuration>
     </plugin>
+           <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>                        
+                        <artifactId>maven-deploy-plugin</artifactId>
+                        <inherited>false</inherited>
+                        <configuration>
+                            <altDeploymentRepository>nexus::default::http://nexus:8081/repository/maven-snapshots/</altDeploymentRepository>
```
Comment (savermyas) [inline pom.xml:72]: Why do you need it here?

File: src/main/resources/git-commit-id.properties [hunk 5]
```diff
@@ -0,0 +1 @@
+git-commit-id-plugin.version=${version}
```
Comment (savermyas) [inline src/main/resources/git-commit-id.properties]: Is this file useful?

**Review by savermyas — CHANGES_REQUESTED** [review]
Address the comments.

File: README.md (hunk 1 above)
Comment (BasileoLuv) [reply README.md]: My mistake. Sorry

File: pom.xml (hunk 4 above)
Comment (BasileoLuv) [reply pom.xml:72]: Because 
"failed: repository element was not specified in the POM inside distributionManagement element or in -DaltDeploymentRepository=id::layout::url parameter"

File: src/main/resources/git-commit-id.properties (hunk 5 above)
Comment (BasileoLuv) [reply src/main/resources/git-commit-id.properties]: This file is useless, so I delete it from the repository.

**Review by savermyas — APPROVED** [review]
